lib/react.dart

```diff
@@ -444,19 +439,20 @@ abstract class Component {
   /// > Due to the release of getSnapshotBeforeUpdate in ReactJS 16,
   /// > componentWillUpdate is no longer the method used to check the state
   /// > and props before a re-render. Both the Component class and
-  /// > componentWillUpdate will be removed in the react.dart 7.0.0 release.
+  /// > componentWillUpdate will be removed alongside Component.
   /// > Use Component2 and Component2.getSnapshotBeforeUpdate instead.
   /// >
-  /// > This will be completely removed when the JS side of it is slated for removal (ReactJS 18 / react.dart 7.0.0)
+  /// > This will be completely removed alongside the Component class.
   void componentWillUpdate(Map nextProps, Map nextState) {}
```

Comment:
I'm a little unclear why certain methods are deprecated, and others are not.

Reply:
I think we deprecated only the ones related to the legacy context, and for the rest deprecating `Component` itself seemed sufficient.